database/instance_db.go

```diff
@@ -12,6 +12,9 @@ type InstanceDB interface {
 	// Get retrives instance by instance hash.
 	Get(hash string) (*instance.Instance, error)
 
+	// GetAllByService retrives all instances of service by service's hash.
+	GetAllByService(hash string) ([]*instance.Instance, error)
```

Comment:
```suggestion
	GetAllByService(serviceHash string) ([]*instance.Instance, error)
```
or 

`GetAllByServiceHash`